### PR TITLE
Profile API

### PR DESCRIPTION
**Pull request by KrystianZybura**
Profile API added along with some other improvements related to MovieList, since it required some corrections for Profile setup working without issues.

Make sure to check the changes I made, and if you have any improvement suggestions, feel free to post them 😎

### CODE REVIEW

File: src/common/MoviesList/index.js [hunk 1]
```diff
@@ -1,58 +1,36 @@
...
-      </MainContainer>
-      <Pagination
-        currentPage={currentPage}
-        totalPages={totalPages()}
-        onFirstPage={() => whichPage(1)}
-        onPrevPage={() => whichPage(currentPage - 1)}
-        onNextPage={() => whichPage(currentPage + 1)}
-        onLastPage={() => whichPage(totalPages())}
-      />
-    </>
```
Comment (Arcziwald) [inline src/common/MoviesList/index.js:55]: moving it is a great idea ;-)

File: src/common/MoviesList/index.js [hunk 2]
```diff
@@ -1,58 +1,36 @@
 import React from "react";
```
Comment (Arcziwald) [inline src/common/MoviesList/index.js]: I think this import is not necessary ;-)

**Review by Arcziwald — COMMENTED** [review]
You did a wonderful job :-D

File: src/common/MoviesList/index.js (hunk 2 above)
Comment (KrystianZybura) [reply src/common/MoviesList/index.js]: Import removed, thanks for the feedback! 💪

File: src/common/MovieTile/index.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ const MovieTile = ({ posterPath, title, year, tag, rating, votes }) => {
         </GenreBox>
         <RatingBox>
           <StarImage></StarImage>
-          <Rating>{rating}</Rating>
+          <Rating>{rating.toFixed(1).replace(".", ",")}</Rating>
```
Comment (WoytekMig) [inline src/common/MovieTile/index.js:43]: Damn, Nice eye ! :D

File: src/common/MoviesList/index.js [hunk 4]
```diff
@@ -1,58 +1,35 @@
...
-        onFirstPage={() => whichPage(1)}
-        onPrevPage={() => whichPage(currentPage - 1)}
-        onNextPage={() => whichPage(currentPage + 1)}
-        onLastPage={() => whichPage(totalPages())}
-      />
-    </>
+    <MainContainer>
+      {moviesData.map((movie) => {
+        const year = new Date(movie.release_date).getFullYear();
+        const roleAndYear = `${movie.character ?? movie.job} (${year})`;
```
Comment (WoytekMig) [inline src/common/MoviesList/index.js:13]: Now this component is fully reusable . Great ! :)

**Review by WoytekMig — APPROVED** [review]
Great job :) Everything seems workig perfectly 👍

**Review by Arcziwald — APPROVED** [review]

**Pull request merged**